securesign/secure-sign-operator
# feat: Add rekor search ui

This pr aims to add the rekor search ui into the operator. Completes: https://github.com/securesign/secure-sign-operator/issues/39

## Testing
1. Usual, cluster, login, build, deploy
2. Create ns `oc create ns trusted-artifact-signer`
3. In your yaml file make sure  `rekor.rekorSearchUI.external = true` to expose the route
4. Apply the yaml file
5. Go to Networking -> Routes -> Rekor-Search-UI
6. When you visit the page you will need to manually input the rekor server url by clicking the gear in the top right
7. You should be able to view info then (sign and verify so there is something there to see ).
